package.json

```diff
@@ -30,18 +30,22 @@
     "path": "0.12.7",
     "rxjs": "6.6.3",
     "svg-icon-sprite": "1.1.1",
+    "ts-jsonapi": "2.1.3",
     "tslib": "2.0.3",
     "typescript": "4.0.5",
     "webdriver-manager": "12.1.7",
     "zone.js": "0.10.3"
   },
   "devDependencies": {
+    "@angular-builders/custom-webpack": "10.0.1",
     "@angular-devkit/build-angular": "0.1100.0",
     "@angular/cli": "11.0.0",
     "@angular/compiler-cli": "11.0.0",
     "@types/jasmine": "3.6.1",
+    "@types/lodash": "^4.14.165",
```

Comment:
@carryall Could we use specific version just like others?

Reply:
✅  Fixed on [e8d6d6e](https://github.com/carryall/survey-challenge/pull/31/commits/e8d6d6eec9da251ee7dead5965815559c3d97062)